Dockerfile

```diff
@@ -17,6 +17,10 @@ RUN pip install pydevd-pycharm~=201.8538.36
 # Install vsdbg for remote C# debugging in Visual Studio and Visual Studio Code
```

Comment:
nit - Should use `--no-cache-dir` for the pip install above

Reply:
Addressed here https://github.com/QuantConnect/Lean/pull/6153